plugins/modules/rabbitmq_user.py

```diff
@@ -103,9 +104,38 @@
     description:
       - C(on_create) will only set the password for newly created users.  C(always) will update passwords if they differ.
     type: str
-    required: false
     default: on_create
     choices: ['on_create', 'always']
+  login_protocol:
+    description:
+      - Specify which TCP/IP protocol will be used.
+    type: str
+    default: http
+    choices: ['http', 'https']
+    aliases: [protocol]
+    version_added: '1.2.0'
+  login_host:
+    description:
+      - Hostname of API.
+    type: str
+    aliases: [host]
+    version_added: '1.2.0'
+  login_port:
+    description:
+      - login_port of access from API.
+    type: str
+    aliases: [login_port]
```

Comment:
```suggestion
```

Reply:
I'll work on your suggestions =]

Reply:
@nrukavkov I need to work on this suggestions.